test/python/circuit/test_parameters.py

```diff
@@ -1178,7 +1178,30 @@ def test_compare_to_value_when_bound(self):
 
         x = Parameter("x")
         bound_expr = x.bind({x: 2.3})
+
         self.assertEqual(bound_expr, 2.3)
+        self.assertTrue(bound_expr < 3.0)
+        self.assertTrue(bound_expr > 1.0)
+        self.assertEqual(abs(bound_expr), 2.3)
```

Comment:
If we keep the comparison operators `__le__` and `__ge__`, we'd need to those here too.